src/services/sanity.js

```diff
@@ -1421,7 +1465,28 @@ function     buildEntityAndTotalQuery(
     }`;
     return query;
 }
+function processMetadata(brand, type, withFilters = false) {
+    const metadataElement = contentMetadata[brand]?.[type];
+    if (!metadataElement) {
+        return null;
+    }
+    const processedData = {
+        type,
+        thumbnailUrl: metadataElement.thumbnailUrl || null,
+        name: metadataElement.name || null,
+        description: metadataElement.description || null
+    };
+
+    if (withFilters) {
+        Object.keys(metadataElement).forEach(key => {
+            if (key !== 'thumbnailUrl' && key !== 'name' && key !== 'description') {
```

Comment:
It's a small suggestion, but I think it would be cleaner to do a "not in array" check instead of multiple "&&"
```suggestion
            if ( !['thumbnailUrl', 'name', 'description'].includes(key) ) {
```